release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -131,6 +131,46 @@ For more information, see xref:../networking/openshift_network_security/configur
 [id="ocp-4-16-storage"]
 === Storage
 
+[id="ocp-4-16-storage-clone-azure-file"]
+==== Volume cloning supported for Azure File (Technology Preview)
+{product-title} 4.16 introduces volume cloning for the Microsoft Azure File Container Storage Interface (CSI) Driver Operator as a Technology Preview feature. Volume cloning duplicates an existing persistent volume (PV) to help protect against data loss in {product-title}.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file[Azure File CSI Driver Operator] and xref:..//storage/container_storage_interface/persistent-storage-csi-cloning.adoc#persistent-storage-csi-cloning[CSI volume cloning].
+
+[id="ocp-4-16-storage-node-expansion-secret"]
+==== Node Expansion Secret is generally available
+The Node Expansion Secret feature allows your cluster to expand storage volumes, even when access to those volumes requires a secret (for example, a credential for accessing a Storage Area Network (SAN) fabric) to perform the node expand operation. {product-title} 4.16 supports this feature as generally available.
+
+[id="ocp-4-16-vsphere-max-snapshot"]
+==== Changing vSphere CSI maximum number of snapshots is generally available
+The default maximum number of snapshots in VMWare vSphere Container Storage Interface (CSI) is 3 per volume. In {product-title} 4.16, you can now change this maximum number of snapshots to a maximum of 32 per volume. You also have granular control of the maximum number of snapshots for vSAN and Virtual Volume datastores. {product-title} 4.16 supports this feature as generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#vsphere-change-max-snapshot_persistent-storage-csi-vsphere[Changing the maximum number of snapshots for vSphere]
```

Comment:
Is the link correct? In preview this leads to vsphere csi driver operator page, but should lead to actual snapshot limit configuration here:

https://github.com/openshift/openshift-docs/pull/74295/files#diff-73b460816ff2db4ab1fb5cb7d16efddc3015860cb41e811acd00131cf7b162c2R8